### ISSUE TITLE
running the code

### ISSUE THREAD

**Issue by NayhaShahid** [finiteautomata/p300speller#5]
I am unable to run the code, could you please provide detailed instructions?

**Comment by finiteautomata** [comment]
Nayla:

This project is just a prototype, and relies on a not-yet-open dataset. The only thing I attempted in this project was P300 detection.

What are you exactly willing to do?

**Comment by NayhaShahid** [comment]
I'm trying to make a p300 speller to run with emotiv epoc+

**Comment by finiteautomata** [comment]
Nayla:

I can't share you my dataset, but check [this notebook](https://github.com/finiteautomata/p300speller/blob/master/notebooks/cnn_all.ipynb) in which I build a classifier using a Convolutional Neural Network.

I use [mne](https://martinos.org/mne/stable/index.html) to load the data from the Emotiv headset.

**Comment by abujbr** [comment]
Hello,

can't we use public (open access) dataset?

**Comment by finiteautomata** [comment]
Yes, you can use some public dataset. I think the model should work in a similar fashion
